README.md

```diff
@@ -182,12 +182,14 @@ cirrus_config.update(**settings)
 *Still uses Google libraries for auth*
 
 ## Building the Documentation
-- `pip install -r dev-requirements.txt`
-- `python docs/create_docs.py`
+- `pipenv install --dev`
+- `pipenv run python docs/create_docs.py`
 - HTML is generated in the `docs/build` folder
 
-## Python 3 Compatibility Notes
-`psutil` doesn't install correctly when doing `pip install -r requirements.txt`
```

Comment:
does this work with pipenv?

Reply:
Oh, I removed this because I didn't find usage in this package.